jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/util/SLAComplianceCell.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.workbench.common.client.util;
+
+import java.util.List;
+
+import javax.enterprise.context.Dependent;
+import javax.inject.Inject;
+
+import org.jboss.errai.common.client.dom.DOMTokenList;
+import org.jboss.errai.common.client.dom.Span;
+import org.jboss.errai.ui.client.local.api.IsElement;
+import org.jboss.errai.ui.shared.api.annotations.DataField;
+import org.jboss.errai.ui.shared.api.annotations.Templated;
+import org.kie.api.runtime.process.ProcessInstance;
+
+import com.google.gwt.cell.client.AbstractCell;
+import com.google.gwt.safehtml.shared.SafeHtmlBuilder;
+
+@Dependent
+@Templated(value = "SLAComplianceCell.html")
+public class SLAComplianceCell extends AbstractCell<Integer> implements IsElement {
+
+    private static final String SPAN_NAME = "sla-compliance";
+    private static final String SLA_STYLE = "label";
+    private static final String SLA_NA_PRESENT_STYLE = "label-default";
+    private static final String SLA_PENDING_PRESENT_STYLE = "label-primary";
+    private static final String SLA_MET_PRESENT_STYLE = "label-success";
+    private static final String SLA_ABORTED_PRESENT_STYLE = "label-warning";
+    private static final String SLA_VIOLATED_PRESENT_STYLE = "label-danger";
+
+    
+    private List<String> descriptions;
+    @Inject
+    @DataField(SPAN_NAME)
+    private Span content;
```

Comment:
To adjust this element to the new API you could use:
`@Inject
    @DataField(SPAN_NAME)
@Named("span")
    private HTMLElement content;`